pkg/proxy/endpoints.go

```diff
@@ -237,6 +239,12 @@ func (ect *EndpointChangeTracker) endpointsToEndpointsMap(endpoints *v1.Endpoint
 		return nil
 	}
 
+	if endpoints.Annotations[isHeadlessServiceAnnotation] == "true" {
```

Comment:
What happens if you get "false" or "TRUE" or "True" or "1" ?

Let's be clear to document either `Only the exact value "true" (as a string) is considered true.` or else `The existence of this annotation is the signal and the value, while conventionally "true" (as a string) is not meaningful."

Reply:
So the idea here is to use this annotation the same way scheduler uses the ``scheduler.alpha.kubernetes.io/critical-pod``, with the simple existence of it representing that this is a Non VIP'ed service, right? Will take a look and improve this!

Thanks for your time :D

Reply:
It's up to you, but that's how I would lean.  The value is "true" by convention, but even setting it to "false" or "" would still be true.

Reply:
So I'm assuming the existence of the Label is enough to ignore it into the informer. 

I've also described this into the const comment, is this enough? Otherwise I can be more descriptive